### PR TITLE
@oxaudo => Deploy new artwork page to a configurable percentage of users

### PR DESCRIPTION
**Pull request by mzikherman**
Once https://github.com/artsy/reaction/pull/1774 is merged, we can update Reaction here and this should be good to go!

This just follows the exact same approach we took for the artist page, so we can adjust how many people see the page (from none, to a full rollout, in increments of 10%) via an ENV var.

### CODE REVIEW

File: src/desktop/apps/artwork/maybe_new_artwork_page.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const { NEW_ARTWORK_PAGE_THRESHOLD } = require("desktop/config.coffee")
+
+module.exports = (_req, res, next) => {
+  // When not set, we serve the existing artist page.
+  if (!NEW_ARTWORK_PAGE_THRESHOLD) {
+    return next()
+  }
+
+  const { NEW_ARTWORK_PAGE } = res.locals.sd
+  if (parseInt(NEW_ARTWORK_PAGE, 10) < NEW_ARTWORK_PAGE_THRESHOLD) {
```
Comment (oxaudo) [inline src/desktop/apps/artwork/maybe_new_artwork_page.ts]: Should we store this random assignment somewhere and reuse it. Unless I'm missing something - the way it's written right now - one can get one page on the first load and an old one in reload, no?

File: src/desktop/apps/artwork/maybe_new_artwork_page.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const { NEW_ARTWORK_PAGE_THRESHOLD } = require("desktop/config.coffee")
+
+module.exports = (_req, res, next) => {
+  // When not set, we serve the existing artist page.
```
[the remark this answers is not in the document]
Comment (mzikherman) [reply src/desktop/apps/artwork/maybe_new_artwork_page.ts]: 🍝

File: src/desktop/apps/artwork/maybe_new_artwork_page.ts (hunk 1 above)
Comment (mzikherman) [reply src/desktop/apps/artwork/maybe_new_artwork_page.ts]: `NEW_ARTWORK_PAGE` is already stored, that's the normal 'random or your existing cookie' value. So everyone gets set from 0-9 randomly (with equal prob.), and that doesn't change.

Then, we configure the threshold, which is the cutoff of who sees it.

Comment (oxaudo) [reply src/desktop/apps/artwork/maybe_new_artwork_page.ts]: Oh... Sorry. 3 hours of sleep today:) I read `parseInt` as random. Thanks for chewing it up for me here...

**Comment by mzikherman** [comment]
@oxaudo https://github.com/artsy/force/pull/3266 was the accompanying Reaction update (already merged), so this is...good to go?!

**Comment by mzikherman** [comment]
Ok, going for it 🤞

**Pull request merged**